utilities/decrypt_data/export_postgres_to_excel_chunks.py

```
@@ -0,0 +1,585 @@
+        Establish connection to PostgreSQL database.
+        
+        Returns:
+            bool: True if connection successful, False otherwise
+        """
+        try:
+            self.connection = psycopg2.connect(self.dsn, cursor_factory=RealDictCursor)
+            self.connection.autocommit = True
+            self.engine = create_engine(self.dsn)
+            logger.info("Successfully connected to database")
+            logger.info("SQLAlchemy engine initialized for pandas")
+            return True
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)